examples/dynamic_behavior/skip_messages.cpp

```diff
@@ -31,12 +31,12 @@ void caf_main(actor_system& system) {
   scoped_actor self{system};
   self->request(serv, std::chrono::seconds(10), ping_atom_v)
     .receive(
-      [&](pong_atom) {
+      [&self, worker](pong_atom) {
         aout(self) << "received response from "
                    << (self->current_sender() == worker ? "worker\n"
                                                         : "server\n");
       },
-      [&](error& err) {
+      [&self, worker](error& err) {
```

Comment:
Not sure if taking self by reference is a best practice.
```suggestion
      [self, worker](error& err) {
```

Reply:
This is also a `scoped_actor`.